### PR TITLE
TST: set style in mpl_toolkits to ease later transition

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

The long term plan is that all test images should use `style='mpl20'` (= `style='default'). To be able to change the default of `image_comparison` to this, it makes sense to change all tests that doesn't explicitly specify style to use the current default.

This PR handles mpl_toolkits.

The comment isn't that explicit as I expect that once the default is changed, one can simply reword it to drop the style. (I do not expect many regenerated images before that happens...)

## PR Checklist

**Documentation and Tests**
- [ ] Has pytest style unit tests (and `pytest` passes)
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] New plotting related features are documented with examples.

**Release Notes**
- [ ] New features are marked with a `.. versionadded::` directive in the docstring and documented in `doc/users/next_whats_new/`
- [ ] API changes are marked with a `.. versionchanged::` directive in the docstring and documented in `doc/api/next_api_changes/`
- [ ] Release notes conform with instructions in  `next_whats_new/README.rst` or `next_api_changes/README.rst`

### CODE REVIEW

**Comment by oscargus** [comment]
Failure seems unrelated (blitting in non-mpl_toolkits).

**Review by QuLogic — APPROVED** [review]

**Review by ksunden — APPROVED** [review]
